scipy/stats/_morestats.py

```diff
@@ -1766,11 +1766,8 @@ def shapiro(x):
     Notes
     -----
     The algorithm used is described in [4]_ but censoring parameters as
-    described are not implemented. For N > 5000 the W test statistic is accurate
-    but the p-value may not be.
-
-    The chance of rejecting the null hypothesis when it is true is close to 5%
-    regardless of sample size.
```

Comment:
I removed this statement because it assumes that the user will be using 0.05 as a threshold for significance without acknowledging the assumption. I believe that `shapiro` approximates p-values for large N with Monte Carlo methods, so I would have modified it to something similar to what I'm writing about `monte_carlo_test` in these examples

> Nonetheless, it can be used to exactly control the rate of false rejections of the null hypothesis [4]_.

but `shapiro`'s p-values are deterministic. If they are "not accurate" and deterministic, I don't see how they can exactly control the false positive rate for a given sample size.